src/app/application/components/route-view/RouteView.vue

```diff
@@ -1,5 +1,6 @@
 <template>
```

Comment:
**Issue**: (leaving this here just so we have a resolvable thread instead of plain comments on the PR)

History navigation is breaking in some cases:

1. Go to https://deploy-preview-1493--kuma-gui.netlify.app/gui/zones/zone-cps
2. Navigate to page 2
3. Use browser history back
4. Use browser history forward

Expected: you end up on page 2

Actual: you stay on page 1 (after step 3)

Note: Repeating steps 3 and 4 after the first style will work, presumably because the history entries has `?page=1` already.

Reply:
Please only look at Mesh Listings, nothing else is complete. If we are ok with MeshListings I can go an roll it out everywhere.